Mathlib/FieldTheory/IntermediateField.lean

```diff
@@ -605,11 +605,63 @@ example {F : IntermediateField K L} {E : IntermediateField F L} : Algebra K E :=
 
 section ExtendScalars
 
+section Subfield
+
+open Subfield
+
+variable {F E E' : Subfield L} (h : F ≤ E) (h' : F ≤ E') {x : L}
+
+/-- If `F ≤ E` are two subfields of `L`, then `E` is also an intermediate field of
+`L / F`. It can be viewed as an inverse to `IntermediateField.toSubfield`. -/
+def _root_.Subfield.extendScalars : IntermediateField F L :=
```

Comment:
Can you close the outer namespace and use `namespace Subfield` for everything?

Reply:
Done.